common/src/main/java/com/wynntils/overlays/PartyMembersOverlay.java

```diff
@@ -78,7 +78,12 @@ protected List<PartyMemberOverlay> getPreviewChildren() {
     }
 
     @SubscribeEvent
-    public void onHadesUserAdded(HadesUserAddedEvent event) {
+    public void onHadesUserAdded(HadesUserEvent.Added event) {
```

Comment:
Since you do the same for both "subevents", you can do like:
```
 public void onHadesUserChange(HadesUserEvent event) {
 ```

Reply:
You can't :). I've thought this is the case! But turns out, event bus does not work like that, at least not now. I have no idea if this is an effect of the LexForge/NeoForge split, Lex have been updating the event bus... Anyways, this only works this way.

Reply:
Oh, really? 🤔 That sounds like a new thing, I'm pretty sure it used to work.